contracts/dtoken/src/views.rs

```diff
@@ -79,7 +88,7 @@ mod tests {
 
         contract.set_total_reserves(200);
 
-        contract
+        contract;
```

Comment:
@dmytro-shash `;` means that this is no more a return value but function should return something